src/Pages/Dashboard/Nav.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from "react";
```

Comment:
Why aren't you using the header component and doing the switch here. You will just be hiding it in the register and login component and add a special class when its not on the home route